async-http-client-backend/zio/src/test/scala/sttp/client/asynchttpclient/zio/SttpBackendStubZioTests.scala

```diff
@@ -26,4 +28,29 @@ class SttpBackendStubZioTests extends AnyFlatSpec with Matchers with ScalaFuture
     runtime.unsafeRun(r).body shouldBe Right("c")
     runtime.unsafeRun(r).body shouldBe Right("a")
   }
+
+  it should "return given web socket response" in {
```

Comment:
This is example of usage of `WebSocketStub` and also stubbing `openWebsocket`. I ought to add similar thing for akka and put both in docs.